benchmark/fixtures/require-cachable.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
 
 const { internalBinding } = require('internal/test/binding');
-const {
-  builtinCategories: { canBeRequired }
-} = internalBinding('builtins');
+let canBeRequired;
+if (process.moduleLoadList.includes('Internal Binding builtins')) {
+  ({builtinCategories: { canBeRequired }} = internalBinding('builtins'));
+} else {
+  ({moduleCategories: { canBeRequired }} = internalBinding('native_module'));
+}
 
+let prefix = process.version < 'v16.' ? '' : 'node:';
 for (const key of canBeRequired) {
-  require(`node:${key}`);
+  require(`${prefix}${key}`);
```

Comment:
Is this necessary? All non-EOL versions of Node.js support the prefix, do we plan to run benchmarks on EOL versions?

Reply:
> do we plan to run benchmarks on EOL versions?

Yes, if the regression goes back to EOL versions (in this case, it did)